McMotdParser.Test/Deserializer/JsonDeserializerTests.cs

```diff
@@ -35,6 +51,7 @@ public void ComplexJsonMotdDeserialize()
             
             List<MotdContent> expect = new List<MotdContent>()
             {
+                new MotdContent{ Color = "#808080", Text = " "},
```

Comment:
This is needed because of the existing text property in the json. 
And you transfer empty string to space. I don't know why...

Reply:
I think it's probably because I didn't parse the text of the root element the first time.

Reply:
this is just mistake maybe?